enhancements/declarative-index-config.md

```diff
@@ -865,34 +865,22 @@ $ tree
 ├── etcd.json
 ├── amqstreams.json
 ```
-#### Editing a package inside an index
+#### Creating new indexes using the config files/Editing a package inside an existing index
 
-A new sub-command `update` will be introduced under `opm index`, which will take a folder containing json representations of packages as it's input along with an existing index image tag, and will replace the edited config files in the index container. 
+A new sub-command `build` will be introduced under `opm index`, which will take a folder containing json representations of packages as it's input, and will build a container image with the folder and the `opm` binary in it. 
```

Comment:
With a `Dockerfile`, `opm index build` becomes little more than UX-sugar unless we plan to support alternate index image formats like OCI.

Related meta-question: will we ever define an index image w/o a runtime component (serve)?

Reply:
I'd rephrase that as `UX-necessity`. I don't think it's reasonable to ask users to track down the correct registry image, write the correct Dockerfile, etc. We hide all of these concerns behind the sub-command, so that users don't have to jump through hoops to build their index image.